glfw/generate-wayland-protocols.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# modified from https://github.com/go-gl/glfw/blob/master/scripts/generate-wayland-protocols.sh
```

Comment:
Instead of using a script like this that we'll have to maintain, etc. I think we should just find out what the right Ubuntu packages are and add them to this list:

https://github.com/hexops/sdk-linux-x86_64/blob/main/fetch.sh#L10-L27

Then we can rerun `fetch.sh` in that repository and commit the result, that way we have a super straightforward way to know we're getting a good version of these headers and we can easily manage other dependencies if GLFW should add them in the future.